cognite/client/data_classes/transformations/common.py

```diff
@@ -148,27 +145,16 @@ def __init__(self, external_id: str = None):
     def __hash__(self) -> int:
         return hash((self.type, self.external_id))
 
-    def __eq__(self, other: Any) -> bool:
-        return isinstance(other, SequenceRows) and hash(other) == hash(self)
-
 
 class DataModelInstances(TransformationDestination):
-    _warning_shown = False
-
-    @classmethod
-    def _show_warning(cls) -> bool:
-        if not cls._warning_shown:
-            warnings.warn(
-                "Feature DataModeStorage is in beta and still in development. Breaking changes can happen in between patch versions."
-            )
-            cls._warning_shown = True
-            return True
-        return False
-
     def __init__(
         self, model_external_id: str = None, space_external_id: str = None, instance_space_external_id: str = None
     ):
-        DataModelInstances._show_warning()
+        warnings.warn(
+            "Feature DataModeStorage is in beta and still in development. "
```

Comment:
```suggestion
            "Feature DataModelStorage is in beta and still in development. "
```

Reply:
Ah, I actually wondered about this one 😆